src/AccessibilityInsights.Extensions.AzureDevOps/Models/Team.cs

```diff
@@ -9,20 +8,20 @@ namespace AccessibilityInsights.Extensions.AzureDevOps.Models
     /// <summary>
     /// Team
     /// </summary>
-    public class Team : AzureDevOpsEntity, ITeam
```

Comment:
>ITeam [](start = 43, length = 5)

Should this PR also remove the now-unused interfaces of ITeam, IProject, IConnectionInfo, etc?

Reply:
No, they haven't been cleaned up fully on the AI-Win side of things :(